src/config/contracts.ts

```diff
@@ -30,6 +31,8 @@ export const contractsAddresses: ContractsAddresses = {
     feeSharingProxy: '0x12B1B0C67d9A771EB5Db7726d23fdc6848fd93ef',
     vestingRegistry: '0x036ab2DB0a3d1574469a4a7E09887Ed76fB56C41',
     XUSDMassetProxy: '0x1440d19436bEeaF8517896bffB957a88EC95a00F',
+    // TODO: update with address once deployed to mainnet
+    rewardManager: '',
```

Comment:
I don't know. If we do have it on mainnet, that should be the subject of a separate ticket and a separate PR.

Reply:
I don't think it should be a separate ticket/PR because you were claiming we're ready for production any day now but how can we be ready for that if we haven't do a single test of it in production?

Reply:
You're absolutely right, I was looking at the wrong thing.